index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+    <head>
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <link rel="stylesheet"  href="css/reset.css" /> 
+     <link rel="stylesheet"  href="css/base.css"/> 
+     <link rel="stylesheet"  href="css/layout.css"/> 
+     <link rel="stylesheet"  href="css/module.css"/> 
+    </head>
+    <body>
+        <div id ="a"><span>a</span></div>
```

Comment:
'span' tags within 'div' tags is a bit much. I recommend one tag per element and try to make it semantic.